src/components/search/Search.jsx

```diff
@@ -49,12 +50,12 @@ const Search = () => {
           />
         )}
         <div className="search-header-wrapper">
-          <SearchHeader containerSize="lg" headerTitle={HEADER_TITLE} />
+          <SearchHeader containerSize="lg" />
```

Comment:
can we link it as well with the feature flag?

Reply:
Done